src/scoringreport.rs

```diff
@@ -0,0 +1,16 @@
+use std::fs;
+use crate::{config::Check};
+
+/// Writes to Scoring Report
+pub fn write_to_scoring_report (file: &String, scored_vector: impl Iterator<Item = Check>) {
+  let mut text = String::from("<ul>\n");
+  for scored in scored_vector {
+    text.push_str(&format!("<li>Check passed: {} - {} points</li>\n", scored.name, scored.points));
+  }
+  text.push_str(&"</ul>\n");
+  let f = fs::write(file, text);
+  match f {
+    Err(error) => println!("Problem opening the file: {:?}", error),
```

Comment:
we're anyhow for error handling so just make this return an `anyhow::Result<()>` and just use `?` with [context](https://docs.rs/anyhow/1.0.38/anyhow/trait.Context.html); make sure to return `Ok(())` at the end

Reply:
also no need to `&` here, `push_str` requires a `&str` so referencing it again is useless